src/OpenTelemetry.AutoInstrumentation.Native/integration_loader.cpp

```diff
@@ -127,31 +127,37 @@ namespace
             return;
         }
 
-        if (!enabledIntegrationNames.empty())
+        // ignore LoggingBuilder, it is technically non-instrumentation
+        // it is using only the same functionality
+        if (name != WStr("LoggingBuilder"))
```

Comment:
Related: https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1310 

Consider leaving a comment referring to the issue.

Reply:
fix 46a0800753ad076f8b78c17594f98aa09bb1986b